### PR TITLE
feat: 🔥 added random status generator

### LINKED ISSUE #112: randStatus
### Description

active, pending, etc.

### Proposed solution

_No response_

### Alternatives considered

_No response_

### Do you want to create a pull request?

No

### PR DESCRIPTION
**Pull request by arulprabakaran**
✅ Closes #112 

## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/ngneat/falso/blob/master/CONTRIBUTING.md#commit
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

```
[ ] Bugfix
[x] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Documentation content changes
[ ] Other... Please describe:
```

## What is the current behavior?

Issue Number: #112 

## Does this PR introduce a breaking change?

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Comment by theryansmee** [comment]
This is failing intermittently because of my tests for fullName. I will raise a separate issue and PR to fix. Until then, retuning should fix it

**Comment by theryansmee** [comment]
I have put a hotfix in for this issue here https://github.com/ngneat/falso/pull/121

File: packages/falso/src/lib/status.ts [hunk 1]
```diff
@@ -0,0 +1,38 @@
+import { FakeOptions, fake } from './core/core';
+import { data } from './status.json';
+
+type Type = 'Project' | 'User Story' | 'Task';
+
+/**
+ * Generate a random status.
+ *
+ * @category TODO
```
Comment (NetanelBasal) [inline packages/falso/src/lib/status.ts:9]:
```suggestion
 * @category general
```

File: packages/falso/src/lib/status.ts [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+ *
+ * @example
+ *
+ * randStatus()
```
Comment (NetanelBasal) [inline packages/falso/src/lib/status.ts:13]: add an example for type please

File: packages/falso/src/lib/status.ts (hunk 1 above)
Comment (arulprabakaran) [reply packages/falso/src/lib/status.ts]: 👍

File: packages/falso/src/lib/status.ts (hunk 2 above)
Comment (arulprabakaran) [reply packages/falso/src/lib/status.ts:13]: 👍

**Pull request merged**